TwitchPlaysAssembly/Src/Commands/GameCommands.cs

```diff
@@ -186,6 +186,20 @@ public static void ListUnsolved(string user, bool isWhisper)
 		}
 	}
 
+	[Command(@"(?:view( *pin)?|(pin *)?view) +(.+)")]
+	public static void ViewPin(string user, bool isWhisper, [Group(1)] bool pin1, [Group(2)] bool pin2, [Group(3)] string viewWhat)
+	{
+		var strings = viewWhat.Split(new[] { ' ', ',', ';' }, StringSplitOptions.RemoveEmptyEntries);
+		var modules = strings.Length == 0 ? null : TwitchGame.Instance.Modules.Where(md => strings.Any(str => str.EqualsIgnoreCase(md.Code))).ToArray();
+		if (modules == null || modules.Length == 0)
+		{
+			IRCConnection.SendMessage($"{user}: no such module.", user, !isWhisper);
```

Comment:
A few things about this message. I think this should include a `@` so the user gets notified. Usually we use a `,` and not a `:` so maybe that should be changed for consistency's sake. And maybe change the message itself to `none of those modules could be found` which is a bit more verbose but I think is a bit clearer about which command was run.